### PR TITLE
Automate several Broken Chain archetype features

### PR DESCRIPTION
**Pull request by pedrogrullada**
Also remove unused tags

### CODE REVIEW

File: packs/feat-effects/effect-bloc-tactics.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                "predicate": [
+                    "bloc-tactics"
+                ],
+                "selector": "deception",
+                "slug": "bloc-tactics-status",
+                "type": "status",
+                "value": 4
+            },
+            {
+                "key": "FlatModifier",
```
Comment (simonward) [inline packs/feat-effects/effect-bloc-tactics.json:41]: Should this have `"hideIfDisabled": true,`?

Comment (pedrogrullada) [inline packs/feat-effects/effect-bloc-tactics.json:41]:
```suggestion
            {
                "hideIfDisabled": true,
                "key": "FlatModifier",
```

File: static/lang/re-en.json [hunk 2]
```diff
@@ -2386,6 +2386,17 @@
                 "Cooldown": "Your eidolon then can't use their Breath Weapon again for the next [[/r 1d4 #rounds]]{1d4 rounds}.",
                 "Scaling": "At 3rd level and every 2 levels thereafter, the damage increases by 1d6."
             },
+            "BrokenChain": {
+                "ReleaseMe": {
+                    "BonusToggle": "Release Me! — Apply melee Strike bonus"
+                },
+                "UltimatumOfLiberation": {
+                    "BonusIncreaseToggle": "Ultimatum of Liberation — Increase save bonus to +3"
```
Comment (simonward) [inline static/lang/re-en.json:2394]: Should this be `"Ultimatum of Liberation — Within 30 feet of origin"`? I could go either way I guess.

Comment (pedrogrullada) [reply static/lang/re-en.json:2394]: I considered something along those lines but I couldn't get something unambiguous enough that also wasn't super long.

**Pull request merged**